File: nim/keep_up_the_hoop.nim
#[
    8 kyu
    Keep up the hoop
    Write a program where Alex can input (n) how many times the hoop goes round
    and it will return him an encouraging message:

    If Alex gets 10 or more hoops, return the string "Great, now move on to tricks".
    If he doesn't get 10 hoops, return the string "Keep at it until you get it".
]#

proc hoopCount*(n: int): string =
  case n
  of n >= 10: return "Great, now move on to tricks"
  of n < 10: return "Keep at it until you get it"

echo(hoopCount(10))
echo(hoopCount(8))

File: python/keep_up_the_hoop.py
# 8 kyu
# Keep up the hoop
# https://www.codewars.com/kata/55cb632c1a5d7b3ad0000145/train/python
import codewars_test as test


def hoop_count(n):
    return "Great, now move on to tricks" if n >= 10 else "Keep at it until you get it"


@test.describe("Fixed Tests")
def fixed_tests():
    @test.it("Basic Test Cases")
    def basic_test_cases():
        test.assert_equals(hoop_count(3), "Keep at it until you get it")
        test.assert_equals(hoop_count(11), "Great, now move on to tricks")
